odoo/addons/base/data/base_country_address_data.xml

```diff
@@ -0,0 +1,238 @@
+        <!-- Partner country specific base view -->
+        <record id="view_partner_address" model="ir.ui.view">
+            <field name="name">res.partner.form</field>
+            <field name="model">res.partner</field>
+            <field name="priority" eval="100"/>
+            <field name="arch" type="xml">
+                <form string="Partner Address">
+                    <field name="is_company" invisible="1"/>
+                    <field name="type" invisible="1"/>
+                    <field name="parent_id" invisible="1"/>
+                    <div class="o_address_format">
+                        <field name="street" placeholder="Street..." class="o_address_street"/>
+                        <field name="street2" placeholder="Street 2..." class="o_address_street"/>
+                        <field name="zip" placeholder="ZIP" class="o_address_zip"/>
+                        <field name="city" placeholder="City" class="o_address_city"/>
+                        <field name="state_id" class="o_address_state" placeholder="State" options='{"no_open": True}'/>
+                        <field name="country_id" placeholder="Country" class="o_address_country" options='{"no_open": True, "no_create": True}'/>
+                    </div>
+                </form>
+            </field>
+        </record>
+
+        <!-- Address Format:
+                street
+                street2
+                City Zip
+                State_name
+                Country_name -->
+        <record id="view_partner_address_street_city_zip_state_country" model="ir.ui.view">
+            <field name="name">view partner address form</field>
+            <field name="model">res.partner</field>
+            <field name="inherit_id" ref="base.view_partner_address"/>
+            <field name="mode">primary</field>
+            <field name="arch" type="xml">
+                <xpath expr="//div[hasclass('o_address_format')]" position="attributes">
+                    <attribute name="class">o_address_format o_zip_city</attribute>
+                </xpath>
+                <xpath expr="//field[@name='zip']" position="replace"/>
+                <xpath expr="//field[@name='city']" position="after">
+                    <field name="zip" placeholder="ZIP" class="o_address_zip"/>
+                </xpath>
+                <xpath expr="//field[@name='state_id']" position="replace"/>
+                <xpath expr="//field[@name='country_id']" position="before">
+                    <div>
+
+                        <field name="state_id" placeholder="state id" options='{"no_open": True}'/>
+                    </div>
+                </xpath>
+            </field>
+        </record>
+
+        <!-- Address Format
+                street
+                street2
+                city
+                state
+                zip
+                Country_name -->
+        <record id="view_partner_address_street_city_state_zip_country" model="ir.ui.view">
```

Comment:
The base partner view have the same sequence so remove this. Also, remove from where you use it.